shared-bin/src/fairings.rs

```diff
@@ -11,16 +11,20 @@ pub fn log_launch() -> impl Fairing {
                 rocket.config().port
             );
 
-            tracing::info!(endpoint = %http_endpoint, "HTTP interface is ready");
+            tracing::info!(target: "http", endpoint = %http_endpoint, "HTTP interface is ready");
         })
     })
 }
 
 /// Attach this fairing to enable logging Rocket HTTP requests
 pub fn log_requests() -> impl Fairing {
-    AdHoc::on_request("Rocket HTTP request", |request, _data| {
+    AdHoc::on_response("Log status code for request", |request, response| {
```

Comment:
❓ Is this meant to remain as `on_request`?

Reply:
Nope, needs to be `on_response` so we can get the status code!

Reply:
You're right, I got confused with the documentation.